servicetest/timingprofiling/simple.c

```diff
@@ -21,6 +20,7 @@
 
 #include <unistd.h>
 
-int main(int argc, char **argv) {
+int main(int argc, char * *argv)
```

Comment:
This should not be a part of this commit right?

Also, if this is how the config wants the code to look, could we configure that so it wants it to look like we want it?